src/ap/resources/META-INF/services/javax.annotation.processing.Processor

```diff
@@ -1 +1,2 @@
-org.spongepowered.plugin.processor.PluginProcessor
+org.spongepowered.processor.plugin.PluginProcessor
```

Comment:
While I understand why you move it, I would prefer if it stays in its original package so [we don't break older versions of SpongeGradle](https://github.com/SpongePowered/SpongeGradle/blob/master/src/main/groovy/org/spongepowered/gradle/plugin/SpongePluginBasePlugin.groovy#L37)